share/instruments/instruments.xml

```diff
@@ -3600,13 +3600,13 @@
                         <program value="22"/> <!--Harmonica-->
                   </Channel>
             </Instrument>
-            <Instrument id="harmonica-d12-g">
+            <Instrument id="harmonica-d10g">
                   <family>harmonicas</family>
                   <trackName>10 Hole Diatonic Harmonica</trackName>
                   <longName>10 Hole G Diatonic Harmonica</longName>
                   <shortName>Harm.</shortName>
                   <dropdownName meaning="tuning">G</dropdownName>
-                  <description>10-hole diatonic harmonica pitched in high G.</description>
+                  <description>10-hole diatonic harmonica pitched in G.</description>
```

Comment:
@oktophonie, I think this description was wrong so I changed it. There is another 10 hole diatonic harmonica that really is the "high G" variant.